### PR TITLE
update gather_tree doc

### PR DESCRIPTION
**Pull request by FrostML**
### PR types

Others

### PR changes

Docs

### Describe

Update `paddle.nn.functional.gather_tree()` doc.
Cn doc: [PaddlePaddle 3174](https://github.com/PaddlePaddle/FluidDoc/pull/3174)

### CODE REVIEW

**Comment by FrostML** [comment]
image
image
image

**Review by guoshengCS — DISMISSED** [review]

**Review by zhiqiu — DISMISSED** [review]
Discussed offline, upgrade for core.ops in next PR.

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -14984,32 +14984,31 @@ def gather_tree(ids, parents):
...
-        Variable: A Tensor with the same shape and data type as :attr:`ids`. \
+        Tensor: A Tensor with the same shape and data type as :attr:`ids`. \
             It contains the full sequences. The sequences are collected from \
             :attr:`ids` by backtracing according to :attr:`parents`.
 
     Examples:
         .. code-block:: python
 
-            import paddle.fluid as fluid
+            import paddle
```
Comment (TCChenlong) [inline python/paddle/fluid/layers/nn.py:15002]: 示例代码的改动同中文

**Review by TCChenlong — APPROVED** [review]
LGTM

**Review by zhiqiu — APPROVED** [review]

**Pull request merged**